pkg/operator/yugabytedb/controller.go

```diff
@@ -175,14 +175,6 @@ func validateClusterSpec(spec yugabytedbv1alpha1.YBClusterSpec) error {
 		return err
 	}
 
-	if &spec.Master.VolumeClaimTemplate == nil {
```

Comment:
Could this possibly be an upgrade check or something?

Reply:
The reason behind removing this, address of the variable is [never nil](https://staticcheck.io/docs/checks#SA4022). So, I just removed this check.